Sources/BraveWallet/WalletStrings.swift

```diff
@@ -257,8 +257,8 @@ extension Strings {
       "wallet.addAccountTitle",
       tableName: "BraveWallet",
       bundle: .strings,
-      value: "Add Account",
-      comment: "The title of the add account screen"
+      value: "Add %@ Account",
+      comment: "The title of the add account screen. '%@' will be placed with a coin type title. For example, 'Add Ethereum Account'. "
```

Comment:
i dont think so. i think when the value changed it will be retranslated. Double check with the README.md under `l10n/tools`, dont think we need to update the key.
cc @iccub to confirm.

Reply:
Resolved with this change request: https://github.com/brave/brave-ios/pull/6075#discussion_r998897468
This key remains now, and a new localization is added for new title.